### ISSUE TITLE
Add more attributes to the tuple of RadiusCheck model

### ISSUE THREAD

**Issue by rohithasrk** [openwisp/django-freeradius#108]
There are very limited attributes as of now for the RadiusCheck model. We could have additional attributes or add a way to use those additional attributes. Like `Simultaneous-Use` etc. We'll have to research a bit more about this and then proceed. This option exists for the RadiusGroupCheck as of now.

**Comment by rohithasrk** [comment]
Closed with #

**Comment by 2stacks** [comment]
Agreed, I was going to work on Simultaneous-Use since I use it often.  I typically use RadGroupCheck but it could be implemented per user in RadCheck.

**Comment by rohithasrk** [comment]
@2stacks I wasn't actually to add restrictions on users using the RadGroupCheck model. Can you share your freeradius config if possible/ tell me how you were able to do it?

**Comment by 2stacks** [comment]
I haven't tested this with django-freeradius and postgresql yet.  This is
from another project I maintain.
https://hub.docker.com/r/2stacks/freeradius/ I'm in the process of updating
to support postgresql.

1.  In 'sites-enabled/default'

accounting {
# radutmp
# sradutmp

        sql
}

session {
# radutmp

sql
}

2. Uncomment Simultaneous Use Checking Queries in '$INCLUDE
${modconfdir}/${.:name}/main/${dialect}/queries.conf'

3. Add to SQL some entries similar to this.

INSERT INTO `radgroupcheck` VALUES
(1,'Monthly','Simultaneous-Use',':=','1'),(2,'Semi-Annually','Simultaneous-Use',':=','2'),(3,'Annually','Simultaneous-Use',':=','5');
INSERT INTO `radusergroup` VALUES ('user1','Monthly',1),('user2','
Semi-Annually',1),('user3','Annually',1);

**Comment by 2stacks** [comment]
@rohithasrk are you working to add implementation of Simultaneous-Use with an extra field in radius profile?  I think I can help if you are busy with something else.

**Comment by rohithasrk** [comment]
@2stacks no. As of now working on the token authentication on the API.

**Comment by nemesisdesign** [comment]
@2stacks @rohithasrk I think currently the best thing to do is to:

1. include the standard or most common freeradius attributes in the default list
2. allow to extend the list with a django setting (and explain it in the docs)

We can do this during the Google Code-In with the help of new contributors.
I want to do the same thing for all the choice options.

**Comment by 2stacks** [comment]
Sounds good.  I most often use "simultaneous-use" attribute but it
would be good to have a way to add to a hard coded list of default
options.
